main.py

```diff
@@ -61,7 +61,7 @@ def read_json() -> dict:
 
 def convert_json_to_html(json_data: dict) -> str:
     containers: list = json_data["sidebar"]["containers"]
-    target: int = sum([1 for i in containers if "global" in i])
+    target: int = next(i + 1 for i, c in enumerate(containers) if "global" in c)
```

Comment:
If no container with "global" is found, this will raise a StopIteration exception. It might be worth considering adding a fallback or error handling for this case.
```suggestion
    try:
        target: int = next(i + 1 for i, c in enumerate(containers) if "global" in c)
    except StopIteration:
        raise ValueError("No container with 'global' found in the sidebar data")
```